modules/bmc/redfish.rb

```diff
@@ -0,0 +1,206 @@
+require 'redfish_client'
+require 'bmc/base'
+require 'bmc/redfish/dell'
+require 'bmc/redfish/hpe'
+
+module Proxy
+  module BMC
+    class Redfish < Base
+      include Proxy::Log
+      include Proxy::Util
+
+      def initialize(args)
+        super
+        load_vendor_overrides
+        @bmc
+      end
+
+      def connect(args = { })
+        # TODO probably verify should be an option.
+        connection = RedfishClient.new("https://#{args[:host]}/", verify: false)
```

Comment:
```suggestion
        connection = RedfishClient.new("https://#{args[:host]}/", verify: args.fetch(:verify, false))
```

Reply:
I've never worked anywhere that deployed valid (non-self-signed) certificates to BMC interfaces, but I don't have a problem making it a config file option.

Reply:
Well, if you have self-signed certificates you can also disable the encryption. That's easier. :-)

Sorry for being a smart-ass here. I know most folks don't use signed certificates for BMC interfaces. But that's so important for security.

Reply:
Not at all, you guys are absolutely right that this shouldn't be default disable, even if I suspect in most deployments it will get disabled.

Reply:
added as config item, default enabled.